app-emulation/fuse/fuse-1.6.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa ao backend-fbcon backend-sdl backend-svga backend-X gpm joystick memlimit png pulseaudio xml"
+
+# Only one UI back-end can be enabled at a time
+# At most one audio driver can be enabled at a time
+REQUIRED_USE="?? ( backend-fbcon backend-sdl backend-svga backend-X )
+	?? ( alsa ao pulseaudio )"
+
+RDEPEND="
+	>=app-emulation/libspectrum-1.5.0
+	dev-libs/glib:2
+	alsa? ( media-libs/alsa-lib )
+	ao? ( media-libs/libao )
+	backend-sdl? ( media-libs/libsdl )
+	backend-svga? ( media-libs/svgalib )
+	backend-X? ( x11-libs/libX11 x11-libs/libXext )
+	!backend-fbcon? ( !backend-sdl? ( !backend-svga? ( !backend-X? ( x11-libs/gtk+:3 ) ) ) )
+	gpm? ( sys-libs/gpm )
+	joystick? ( !backend-sdl? ( media-libs/libjsw ) )
+	png? ( media-libs/libpng:0= sys-libs/zlib )
+	pulseaudio? ( media-sound/pulseaudio )
+	xml? ( dev-libs/libxml2:2 )"
+DEPEND="${RDEPEND}
+	backend-fbcon? ( virtual/linux-sources )
+	dev-lang/perl
+	virtual/pkgconfig"
```

Comment:
Please put the unconditional deps above conditional deps.
Well, pkgconfig belongs to `BDEPEND` with EAPI-7 and most likely `perl` does too.

Reply:
Ordering of unconditional and conditional dependencies: Please reorder the dependencies in https://devmanual.gentoo.org/general-concepts/dependencies/index.html in section "USE-Conditional Dependencies".

Reply:
Sure, you could make a [PR](https://github.com/gentoo/devmanual/pulls) if you're interested.

Reply:
In fact, I've gone ahead and done [this](https://github.com/gentoo/devmanual/pull/197). Let me know if you have any feedback, thanks!

Reply:
I have made a final decision to step out of maintaining any Gentoo packages. Sorry.

Reply:
I'm sorry to hear that. I'll retire you from your packages.